server/modules/surveyRdb/manager/surveyRdbManager.js

```diff
@@ -79,6 +85,42 @@ export const fetchViewData = async (params) => {
   return result
 }
 
+export const fetchEntitiesDataToCsvFiles = async ({ surveyId }, client) => {
+  const survey = await SurveyManager.fetchSurveyAndNodeDefsBySurveyId(surveyId, null, true, false, false, false, client)
+
+  const surveyInfo = Survey.getSurveyInfo(survey)
+  const surveyName = Survey.getName(surveyInfo)
+
+  const exportDataFolderName = `${surveyName}_export_${DateUtils.nowFormatDefault()}`
+  const dir = FileUtils.join(ProcessUtils.ENV.tempFolder, exportDataFolderName)
+  await FileUtils.rmdir(dir)
+  await FileUtils.mkdir(dir)
+
+  const entities = Survey.getNodeDefsArray(survey).filter((node) => NodeDef.isEntity(node) || NodeDef.isMultiple(node))
```

Comment:
can you please rename "entities" in just "nodeDefs" since even multiple attributes are exported as well?